client/src/pages/ServiceCalendar/ServiceCalendar.tsx

```diff
@@ -1,7 +1,29 @@
-import { Typography } from '@mui/material';
+import * as React from 'react';
+import dayjs, { Dayjs } from 'dayjs';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { StaticDateTimePicker } from '@mui/x-date-pickers/StaticDateTimePicker';
 import useAuth from '../../hooks/useAuth';
 
+
 export default function ServiceCalendar() {
-  useAuth();
-  return <Typography variant="h2">Service Calendar</Typography>;
-}
+    const [value, setValue] = React.useState<Dayjs | null>(dayjs('2022-04-17'));
+    console.log(value);
+    useAuth();
+    
+  return (
+
+    <LocalizationProvider dateAdapter={AdapterDayjs} >
+       <StaticDateTimePicker orientation="landscape"
+       value={value}
+       onChange={(newValue) => setValue(newValue)}
+       slotProps={{
+        actionBar: {
+            actions:["today","accept","cancel"],
+           },
+         }} 
+       />
+       
+    </LocalizationProvider>
+  );
+}   
```

Comment:
please resolve this 'no new line at end of file'.

if you are using VSCode, the easiest way to resolve this and guarantee it wont happen in the future is by:

1. Go to settings
2. search and enable 'Insert Final NewLine'
3. Go to the file in question and add and remove the last line, you should see VSCode forcing a new line at the end.

more on this inconvenience [here](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3)